### DISCUSSION TITLE
باز کردن اتوماتیک پاکت

### DISCUSSION THREAD

**Discussion by momarezz** [kianahmadian/python-bale-bot#93] [Q&A] [answered]
چگونه یه کاری بکنیم که وقتی در گروهمان پاکت هدیه فرستادند ان را بله خودکار باز کند ؟

**Comment by kian-ahmadian** [comment, marked as answer]
سلام عزیز وقت بخیر
این موضوع، ارتباطی به این پروژه و کتابخانه ندارد؛ به این دلیل که پاسخ سوال شما با ایجاد اصطلاحا self bot انجام خواهد شد
